packages/core/src/lib/__tests__/modal.spec.ts

```diff
@@ -1,38 +1,71 @@
-import { describe, it, expect, vi } from 'vitest';
-import { render, fireEvent } from '@testing-library/svelte';
+import { describe, it, expect, afterEach, beforeEach } from 'vitest';
+import { render, fireEvent, screen, within } from '@testing-library/svelte';
+import userEvent from '@testing-library/user-event';
 import Modal from '../modal.svelte';
+import { writable } from 'svelte/store';
 
 describe('Modal', () => {
-  it('should emit close event when close icon button is clicked', async () => {
-    const { getByTitle, component } = render(Modal, {
-      open: true,
+  let isOpen = writable(true);
+  let currentValue = true;
+
+  beforeEach(() => {
+    isOpen = writable(true);
+    currentValue = true;
```

Comment:
You can use `get` from `svelte/store` to access the current store value

```ts
import { get, writable } from 'svelte/store';

// ...

isOpen = writable(true);
console.log(get(isOpen)); // logs `true`
```

Reply:
That's so much simpler, tyty!